pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,23 @@
+{ cmake, fetchFromGitHub, lib, pkgconfig, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "fastfetch";
+  version = "1.5.3";
+
+  nativeBuildInputs = [ cmake pkgconfig ];
```

Comment:
Canonical src is written before and passed.

Reply:
Done